lib/elasticsearch_query/filter_formatter.rb

```diff
@@ -4,12 +4,17 @@ module FilterFormatter
     autoload :Custom,'elasticsearch_query/filter_formatter/custom'
     autoload :Match, 'elasticsearch_query/filter_formatter/match'
     autoload :Range, 'elasticsearch_query/filter_formatter/range'
+    autoload :Terms, 'elasticsearch_query/filter_formatter/terms'
 
     class << self
       def formatter_for( value )
         case value
+        when String && /\.\./
+          FilterFormatter::Range
+        when String && /,/
+          FilterFormatter::Terms
```

Comment:
- Whoa, I didn't know that regex worked with ===
- You should compile these regexes separately to avoid re-initializing them every time this function is called

Really, the only way you can correctly know whether to use a terms or a match query is to look at the mapping for the field and determine if it's text or keyword (this is how my last company did it).  But, for the purposes of this gem, this is fine.